microsoft-edge/webview2/release-notes.md

```diff
@@ -144,6 +144,372 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
 <!-- end of Jan 2024 Prerelease SDK -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2357-prerelease
+
+Release Date: January 30, 2024
+
+[NuGet package for WebView2 SDK 1.0.2357-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2357-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 122.0.2363.0 or higher.<!-- todo: differs from prerelease # this time? -->

```

Comment:
```suggestion
For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 122.0.2357.0 or higher.<!-- todo: differs from prerelease # this time? -->

```

@victorhuangwq Should this be 2357 instead?

Reply:
```
For full API compatibility, this Prerelease version of the WebView2 SDK requires 
the WebView2 Runtime that ships with Microsoft Edge version 122.0.2357.0 or higher.
```

Reply:
yes this should be 2357.

Reply:
pushed commit, building...